README.md

```diff
@@ -10,6 +10,9 @@ Please read the [paper](http://okmij.org/ftp/Haskell/extensible/exteff.pdf) for
 
   * Effects can be added, removed, and interwoven without changes to code not
     dealing with those effects.
+  * You get a better decoupling between an implementation and its interface.
+    For example whether or not you rely on IO for implementing a certain effect or not,
+    is completely transparent to the user.
```

Comment:
@eskimor i'm not sure i follow. isn't this always transparent given haskell's type system whether or not we use `extensible-effects`?